### ISSUE TITLE
Require space after semicolon in object destructuring

### ISSUE THREAD

**Issue by brzpegasus** [dockyard/ember-suave#56]
```js
// bad
const { String:EmberString } = Ember;

// good
const { String: EmberString } = Ember; 
```

**Comment by rwjblue** [comment]
Should be 'after colon' (not semicolon).

**Comment by brzpegasus** [comment]
@rwjblue
> Should be 'after colon' (not semicolon).

lol Oh yeah, woops! Thanks for catching that.

**Comment by hzoo** [comment]
This would be simple rule to add in JSCS itself

**Comment by rwjblue** [comment]
Sure, but then we have to type so many extra spaces :stuck_out_tongue_closed_eyes:

**Comment by hzoo** [comment]
I guess you could not type them and then autofix it all haha

**Comment by rwjblue** [comment]
Absolutely, and I was really just joking around.  In my opinion getting rules upstream (when they make sense) is definitely the best path forward for us.

**Comment by hzoo** [comment]
I'd be ok with leaving the issue here - I did make https://github.com/jscs-dev/node-jscs/issues/1390 a while ago. Should be easy to port though - just need to add some more test cases and docs
